### DISCUSSION TITLE
Projects Beta: Moving a task/issue to "Done" does not close the linked issue

### DISCUSSION THREAD

**Discussion by jontingvold** [community/community#12810] [Projects and Issues]
When you move a task/issue to the "Done" column in the new projects boards/tables, the linked issue (in the issues tab) is not closed. This is quite confusing.

**Comment by naquiuddin** [comment]
True, but the reverse action does work. I got confused too initially.

**Comment by KurtisStirling** [comment]
Should work both ways...

**Comment by daniel-gato** [comment]
same here

**Comment by gerardcobas** [comment]
+1

**Comment by YuezhenQin** [comment]
+1
